gaphor/ui/filemanager.py

```diff
@@ -326,6 +359,31 @@ def response(answer):
             confirm_shutdown()
 
 
+def merge_conflict_dialog(window: Gtk.Window, filename, handler) -> None:
+    dialog = Adw.MessageDialog.new(
+        window,
+        gettext("Resolve merge conflict in model “{filename}”.").format(
+            filename=filename.name
+        ),
+    )
+    dialog.set_body(
+        gettext(
+            "It looks like the model contains a merge conflict. Do you want to resolve it?"
```

Comment:
The model you are opening contains a merge conflict. Do you want to open the current change or the incoming change to the model?

Reply:
That's better indeed.